specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/ddosCustomPolicy.json

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
Please put this file in `Microsoft.Network/stable/2018-10-01`, not `2018-08-01`

Reply:
Don't see that it's done...

Reply:
done now :)